js/common.js

```diff
@@ -175,6 +175,43 @@ $(document).ready( function() {
 		});
 	}
 
+	/**
+	 * Enable range selection for checkboxes, inside a container having class "checkbox-range-selection"
+	 * Assumes the bootstrap/ace styled checkboxes:
+	 *	<div class="checkbox">
+	 *		<label>
+	 *			<input type="checkbox" class="ace">
+	 *			<span class="lbl"></span>
+	 *		</label>
+	 *	</div>
+	 */
+	$('.checkbox-range-selection').on('click', 'div.checkbox label', function (e) {
+		var jcontainer = $(this).closest('.checkbox-range-selection');
+		var last_clicked = jcontainer.data('checkbox-range-last-clicked');
+		if (!last_clicked) {
+			last_clicked = this;
+		}
+		if (e.shiftKey) {
+			// Because shift-click is triggered in a label/span, some browsers
+			// will activate a text selection. Remove text selection.
+			window.getSelection().removeAllRanges();
```

Comment:
Confirmed.
It works quite smooth on Firefox und Chrome.
On Safari, IE and Edge I get a flickering, caused by selecting / unselecting the whole text in row